packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/proxies/mira/DKGProxy.java

```diff
@@ -0,0 +1,25 @@
+package software.uncharted.terarium.hmiserver.proxies.mira;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+import software.uncharted.terarium.hmiserver.models.mira.DKG;
+
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+
+@RegisterRestClient(configKey = "mira-dkg")
+@Produces(MediaType.APPLICATION_JSON)
```

Comment:
For the exception handling in the resource classes to work you will need to annotate this class with `@RegisterProvider(HmiResponseExceptionMapper.class)`. That will pass the responses through the HmiResponseExceptionMapper class, which converts 400 and 500 series errors to `ResponseRuntimeException`